src/crypto/tls/auth.go

```diff
@@ -340,6 +294,7 @@ func selectSignatureScheme(vers uint16, c *Certificate, peerAlgs []SignatureSche
 		// can assume that it supports SHA1. See RFC 5246, Section 7.4.1.4.1.
 		peerAlgs = []SignatureScheme{PKCS1WithSHA1, ECDSAWithSHA1}
 	}
+
```

Comment:
nit: We should minimize changes to upstream files where possible, in order reduce rebase conflicts.